src/test/java/com/wikia/webdriver/common/properties/Credentials.java

```diff
@@ -236,6 +239,9 @@ public Credentials() {
     userNameGoSearchPreferred = XMLReader.getValue(file, "ci.user.goSearchPreferredUser.username");
     passwordGoSearchPreferred = XMLReader.getValue(file, "ci.user.goSearchPreferredUser.password");
 
+    userNameComments = XMLReader.getValue(file, "ci.user.comments.username");
```

Comment:
no need to add this, cause you have already ```COMMENTS_REGULAR_USER``` in Users that will provide you with both username and password for this user